nixos/modules/services/misc/paperless.nix

```diff
@@ -71,7 +71,8 @@ let
     ProtectHostname = true;
     # Would re-mount paths ignored by temporary root
     #ProtectSystem = "strict";
-    ProtectControlGroups = true;
+    # Broke with systemd 254
+    #ProtectControlGroups = true;
```

Comment:
This is likely the wrong fix to apply here. The acutal issue, according to my conversation with upstream systemd (https://github.com/systemd/systemd/issues/29449) is that `/run` doesn't exist anymore (and that our `TemporaryFilesystem=/:ro` cargo culting is just wrong given that we have `ProtectSystem=` available).

Reply:
Do you have an opinion whether to use `ProtectSystem=strict` with `ReadWritePaths=` or just `ProtectSystem = true`?

Reply:
I think we can (as much as we can) stick with `ProtectSystem=strict` and provide the `ReadWritePaths` (or the bind equivalent) as much as possible. Not defining every path and letting systemd take care of the "system protection" (in the case of `ProtectSystem`) is likely the better way going forward. It is more upgradeable in terms of security settings than what we've done with defining e.g. `/` as `ro` etc..